src/components/GetInspiredFlyer.svelte

```diff
@@ -0,0 +1,187 @@
+<script>
+  import { calendarIcon, clockIcon } from '@/images/icons';
+  import { Icon } from './UI';
+
+  import LogoWithText from '../images/logo_with_text.svg';
+  import { lineIcon } from '../images/icons';
+
+  export let section = '';
+  export let title = '';
+  export let languageAbbreviation = 'EN';
+  export let date = '';
+  export let time = '';
+  export let sponsor1 = null;
+  export let sponsor2 = null;
+</script>
+
+<div class="getInspired">
+  <div class="left">
+    <div class="section">
+      <div class="icon">
+        <Icon icon={lineIcon} />
+      </div>
+      <div class="text">
+        <p>{section}</p>
+      </div>
+    </div>
+    <div class="title">
+      <h2>
+        {@html title}
+        <span class="badge">{languageAbbreviation}</span>
+      </h2>
+    </div>
+    <div class="when">
+      <div class="date">
+        <div class="icon">
+          <Icon icon={calendarIcon} />
+        </div>
+        <p class="dateText">
+          {date}
+        </p>
+      </div>
+      <div class="time">
+        <div class="icon">
+          <Icon icon={clockIcon} />
+        </div>
+        <p class="dateText">
+          {time}
+        </p>
+      </div>
+    </div>
+    <div class="spacer" />
+    <div class="getInspiredLogo spacingRight">
+      {@html LogoWithText}
+    </div>
+    <div class="getInspiredPartners spacingRight">
+      <div class="getInspiredPartnersItem">
+        {@html sponsor1}
+      </div>
+      <div class="getInspiredPartnersItem">
+        {@html sponsor2}
+      </div>
+    </div>
+  </div>
+  <figure class="right getInspiredImg">
+    <img src="/images/mapwithperson.png" alt="person with map" />
+  </figure>
+</div>
+
+<style>
+  .getInspired {
+    background-color: var(--color-yellow);
+    padding-left: 8rem;
+    display: flex;
+  }
+
+  .left {
+    display: flex;
+    flex-direction: column;
+    margin-top: 8rem;
+  }
+
+  .section {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    padding-bottom: 1rem;
+  }
+
+  .section .icon {
+    width: 2rem;
+    height: 1.5rem;
+  }
+
+  .section P {
+    padding-left: 1rem;
+    font-weight: bold;
+    text-transform: uppercase;
+    font-size: 2rem;
+  }
+
+  .getInspired h2 {
+    color: #fff;
+    line-height: 5rem;
+    font-size: 3.5rem;
+    font-weight: bold;
+    margin-bottom: 30px;
```

Comment:
Let's use `rem` here for consistency
```suggestion
    margin-bottom: 3rem;
```